learning/step_by_step/instancing.rst

```diff
@@ -3,109 +3,113 @@
 Instancing
 ==========
 
-Rationale
+Introduction
 ---------
 
-Having a scene and throwing nodes into it might work for small projects, but as
-a project grows, you will naturally add more and more nodes and it can quickly
-become unmanageable. To solve this, Godot allows a project to be separated into
-several scenes. This, however, does not work the same way as in other game
-engines. In fact, it's quite different, so please do not skip this tutorial!
+Creating a single scene and adding nodes into it might work for small
+projects, but as a project grows in size and complexity, the number of nodes
+can quickly become unmanageable. To address this, Godot allows a project
+to be separated into any number scenes. This provides you with a powerful
+tool to organize the different components of your game.
 
-To recap: A scene is a collection of nodes organized as a tree, where
-they can have only one single node as the tree root.
+In :ref:`doc_scenes_and_nodes` you learned that a scene is a collection of
+nodes organized in a tree structure, with a single node as the tree root.
 
 .. image:: img/tree.png
 
-Recall that a scene can be created and saved to disk. You can create and save
-as many scenes as you desire.
+You can create as many scenes as you like and save them to disk. Scenes 
+saved in this manner are called "Packed Scenes" and have a ``.tscn`` filename
```

Comment:
Maybe also mention the binary `.scn` filetype here, in parentheses or something?

Reply:
Maybe, although it may be a bit out of scope for an introductory doc. It might be beter to link to the "Resources" doc which has details on PackedScene and saving/loading from disk.